src/connection.c

```diff
@@ -102,13 +102,21 @@ static void connection_set_state(CONNECTION_HANDLE connection, CONNECTION_STATE
     }
 }
 
+// This callback usage needs to be either verified and commented or integrated into 
```

Comment:
Maybe you should write // TODO: ...
That helps track what is pending.
Might need to get Dan's opinion on this as well.

Reply:
IF you end up adding the "TODO:" then add it everywhere you made such comment.

Reply:
Could you file an issue in GitHub also so it gets tracked? Thanks!

---
In reply to: [162208823](https://github.com/Azure/azure-uamqp-c/pull/203#discussion_r162208823) [](ancestors = 162208823)

Reply:
One issue for all the occurrences should be enough

---
In reply to: [162209233](https://github.com/Azure/azure-uamqp-c/pull/203#discussion_r162209233) [](ancestors = 162209233,162208823)